README.md

```diff
@@ -11,18 +11,38 @@
 
 ## 3<sup>rd</sup> Party SDK Support
 
- * [VRTK.Unity.SteamVR]
- * [VRTK.Unity.OculusUtilities]
+  * [VRTK.Unity.SteamVR]
+  * [VRTK.Unity.OculusUtilities]
 
 ## Introduction
 
-VRTK.Unity.Core is a core collection of design patterns for Unity
-that can be beneficial in VR development.
+VRTK.Unity.Core is a core collection of design patterns for [Unity3d] that can be beneficial in VR development.
+
+## Getting Started
+
+### Setting up a project
+
+  * Create a new project using the 3D Template or open an existing project in [Unity3d] 2018.1 or above.
+  * Update the project `Scripting Runtime Version` to `.NET 4.x Equivalent`.
```

Comment:
Starting with 2018.3 this is actually the default and thus we'd need no instructions for this at all... I don't think we want to decide on only supporting 2018.3 already, though. 🤔

The project settings are also in their own standalone window starting with that version, so those instructions may not match perfectly anymore.

Reply:
These instructions are based on 2018.1 as there's no conclusion on the 2018.3 stuff yet, so if that becomes concrete these can change in the future.